spec/support/database_cleaner.rb

```diff
@@ -1,16 +1,6 @@
 RSpec.configure do |config|
-  config.before(:suite) do
+  config.before(:each) do
     DatabaseCleaner.clean_with(:truncation)
     Rails.application.load_seed
```

Comment:
But you always expect the seed to be there. But we can implement the changes @siddhantbajaj did in #1297 without breaking heroku, so it would only be run before each test suite.